src/loaders/genome_collection/compute_genome_attribs.py

```diff
@@ -281,6 +284,24 @@ def _get_r_list_element(r_list, element_name):
     return r_list[pos]
 
 
+def _r_table_to_df(r_table):
```

Comment:
I'm not well versed enough in R or rpy2 to effectively review this, so I'll just have to assume it works

Reply:
for `trait_counts_atgranularity` it seems to work fine.

Reply:
:+1: